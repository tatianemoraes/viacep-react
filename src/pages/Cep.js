import React from 'react';
import Header from '../components/Header';
import FindCep from '../components/FindCep';
import Address from '../components/Address';

export default function Cep() {
  return (
    <>
      <Header /> 
      <FindCep />
      <Address />
    </>
  )
}
