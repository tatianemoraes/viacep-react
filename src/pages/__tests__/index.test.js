import React from 'react';
import renderer from 'react-test-renderer';
import Cep from '../Cep';
import FindCepContextProvider from '../../contexts/useFindCepContext';

it('renders correctly', () => {
  const tree = renderer
    .create(
    <FindCepContextProvider>
      <Cep />
    </FindCepContextProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});