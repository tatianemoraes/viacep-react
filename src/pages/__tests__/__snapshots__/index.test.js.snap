// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="sc-bdnxRM kGtXFS"
  >
    <h1>
      Busca de CEP
    </h1>
  </div>,
  <div
    className="sc-gtsrHT jsIzZx"
  >
    <h2>
      Consulta de CEP
    </h2>
    <div>
      <input
        name="find"
        onChange={[Function]}
        placeholder="Digite seu CEP"
        type="text"
        value=""
      />
      <button
        onClick={[Function]}
      >
        OK
      </button>
    </div>
  </div>,
  <div
    className="sc-hKFxyN ppiEE"
  >
    <div
      className="address-response"
    >
      <div>
        <label
          htmlFor="city"
        >
          Cidade:
        </label>
        <input
          className="input-city"
          disabled={true}
          id="city"
          type="text"
          value=""
        />
      </div>
      <div>
        <label
          htmlFor="uf"
        >
          UF:
        </label>
        <input
          className="input-uf"
          disabled={true}
          id="uf"
          type="text"
          value=""
        />
      </div>
      <div>
        <label
          htmlFor="street"
        >
          Logradouro:
        </label>
        <input
          className="input-street"
          disabled={true}
          id="street"
          type="text"
          value=""
        />
      </div>
      <div>
        <label
          htmlFor="neighborhood"
        >
          Bairro:
        </label>
        <input
          className="input-neighborhood"
          disabled={true}
          id="neighborhood"
          type="text"
          value=""
        />
      </div>
      <div
        className="sc-dlnjwi hILstj"
      >
        <button
          onClick={[Function]}
        >
          CLEAR
        </button>
      </div>
    </div>
  </div>,
]
`;
